lib/plugins/aws/deploy/compile/events/apiGateway/lib/permissions.js

```diff
@@ -42,14 +19,39 @@ module.exports = {
           }
         `;
 
-          const permissionLogicalId = `${normalizedMethod}PermissionApigEvent${this
-            .resourcePaths.indexOf(path)}`;
+          const permissionLogicalId = `${functionName}ApigPermmission`;
```

Comment:
Is there a need to add a number at the end of the name (can we have two permissions here?)